src/components/application_manager/src/application_manager_impl.cc

```diff
@@ -1338,8 +1337,9 @@ void ApplicationManagerImpl::StopNaviService(
   app->StopStreaming(service_type);
 }
 
-// DEPRECATED
-bool ApplicationManagerImpl::OnServiceStartedCallback(
+#pragma GCC diagnostic push
```

Comment:
@jacobkeeler why this pragma required here ? Is this method is especial?

Reply:
This is a deprecated method which uses another deprecated method, the pragma prevent a warning being generated because of this.

Reply:
@jacobkeeler  such knowledge should be shared somewhere.